index.html

```diff
@@ -135,8 +135,23 @@
-    <p>In the absence of HTTP [[!RFC7231]] Link headers, subscribers MAY fall back to other methods to discover the hub(s) and the canonical URI of the topic. If the topic is an XML based feed, it MAY use embedded link elements as described in Appendix B of Web Linking [[!RFC5988]]. Similarly, for HTML pages, it MAY use embedded link elements as described in Appendix A of Web Linking [[!RFC5988]]. Finally, publishers MAY also use the Well-Known Uniform Resource Identifiers [[!RFC5785]] .host-meta to include the &lt;Link&gt; element with rel="hub".</p>
+    <p>The discovery mechanism aims at identifying at least 2 URLs.</p>
+    <ul>
+      <li>The exact URL of the hub(s) designated by the publisher. If more than one URL is specified, it is expected that the publisher pings each of these URLs, so the subscriber may subscribe to one or more of these.</li>
+      <li>The canonical URL for the topic to which subscribers are expected to use for subscriptions.</li>
+    </ul>
+    <p>The protocol currently supports the following discovery mechanisms. Publishers MUST implement at least one of them:</p>
+    <ul>
+      <li>Link Headers [[!RFC5988]]: the publisher SHOULD include at least one Link Header [[!RFC5988]] with <samp>rel=hub</samp> (a hub link header) as well as exactly one Link Header [[!RFC5988]] with <samp>rel=self</samp> (the self link header)</li>
+      <li>If the topic is an XML based feed, publishers SHOULD use embedded link elements as described in Appendix B of Web Linking [[!RFC5988]]. Similarly, for HTML pages, publishers SHOULD use embedded link elements as described in Appendix A of Web Linking [[!RFC5988]]</li>
+      <li class="warning">Finally, publishers MAY also use the Well-Known Uniform Resource Identifiers [[!RFC5785]] <samp>.host-meta</samp> to include the &lt;Link&gt; element with rel="hub". However, please note that this mechanism is currently at risk and may be deprecated. </li>
+    </ul>
+    <p>When perfoming discovery, subscribers MUST implement all three discovery mechanisms but should respect the following order.</p>
+    <ol>
+      <li>Issue a GET or HEAD request to retrieve the topic. Subscribers MUST check for HTTP Link headers first</li>
+      <li>In the absence of HTTP Link headers, and if the topic is an XML based feed or an HTML page subscribers MUST check for embedded link elements.</li>
+      <li class="warning">In the absence of both HTTP Link headers and embedded link elements, subscribers MUST look for Well-Known Uniform Resource Identifiers [[!RFC5785]] <samp>.host-meta</samp> to include the &lt;Link&gt; element with rel="hub"</li>
```

Comment:
host-meta is defined in https://tools.ietf.org/html/rfc6415#section-6.1 and not in RFC 5785